src/main/resources/db/changelog/changes/03-create-roles-table.yaml

```diff
@@ -0,0 +1,21 @@
+databaseChangeLog:
```

Comment:
According to the task description you should

Add Liquibase changesets to insert roles into DB and assign role for existing users. Be sure you have at least one user in DB with role admin